kasa/iot/modules/ledmodule.py

```diff
@@ -0,0 +1,32 @@
+"""Module for led controls."""
+
+from __future__ import annotations
+
+from ...interfaces.ledinterface import LedInterface
```

Comment:
I'm wondering if we the interfaces should be just named w/o `Interface` as part of their name where the implementation is done in `*Module`s? I.e., `Led`, `LightEffects`, etc. Not a biggie though, so do as you wish.

Reply:
That's done